source/github-helpers/index.ts

```diff
@@ -184,3 +184,7 @@ export function scrollIntoViewIfNeeded(element: Element): void {
 	// @ts-expect-error No Firefox support https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoViewIfNeeded
 	(element.scrollIntoViewIfNeeded ?? element.scrollIntoView).call(element);
 }
+
+export function getDiscussionAuthor(): string | undefined {
```

Comment:
This should be called `conversation`; "discussion" in classnames is pre-"GitHub Discussions" terminology. 

Just to double-check, this selector doesn't work on actual discussions like https://github.com/parcel-bundler/parcel/discussions/9828